src/clj/game/cards-operations.clj

```diff
@@ -212,6 +212,15 @@
               " [Credits]")
     :effect (effect (gain :credit (count (filter #(not (empty? %)) (map #(:content (second %)) (get-remotes @state))))))}
 
+   "Election Day"
+   {:req (req (->> (get-in @state [:corp :hand])
+                   (filter #(not (= (:cid %) (:cid card))))
+                   (count)
+                   (pos?)))
+    :msg (msg "to trash all cards in HQ and draw 5 cards")
```

Comment:
This `:msg` will yield:  "(Corp name) uses Election Day **to to** trash all cards in HQ and draw 5 cards"